articles/quickstart/native/ios-objc/dashboard-default.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Getting Started
+description: Short Introduction to the Auth0 iOS Objective-C Quickstarts.
+budicon: 715
+---
+
+<%= include('../../../_includes/_package', {
+  org: 'auth0-samples',
+  repo: 'auth0-ios-objc-sample',
+  path: '00-Login',
+  requirements: [
+    'CocoaPods 1.2.1',
+    'Version 8.3.2 (8E2002)',
+    'iPhone 7 - iOS 10.3 (14E269)'
+  ]
+}) %>
+
+> It is recommend in order to make the most of all the features in the [Auth0.swift](https://github.com/auth0/Auth0.swift) toolkit to start with the [iOS Swift QuickStart](/quickstart/native/ios-swift)
+
+This multistep quickstart guide will walk you through managing authentication in your iOS Objective-C apps with Auth0.
+
+## About the Sample Applications
+
+Each step within this quickstart guide (starting with **Login**) contains a downloadable sample which demonstrates the topic being covered. Within each of these samples, you will find an appropriate application to demonstrate the features discussed in the chapter.
+
+## Auth0 Dependencies
+
+Each tutorial will require you to use either the [Auth0.swift](https://github.com/auth0/Auth0.swift) toolkit, the [Lock](https://github.com/auth0/Lock.swift) widget or both.
```

Comment:
As LOCK articles are removed from the index, i would just talk about auth0 dependency here.